### ISSUE TITLE
Using arrows and dots together lets user scroll last slide to first position if items > 1

### ISSUE THREAD

**Issue by tpinne** [YIZHUANG/react-multi-carousel#32]
**Describe the bug**
If arrows and dots are enabled together with more than 1 item visible on a breakpoint, the next arrow disappears as soon as the last slide enters the view. Which is totally correct and expected.

The dot indicator then shows the currently first visible slide as active. Which is also sort of correct. But if the user now clicks on one of dots right next to the active one, the carousel slides the corresponding slide to the first position leaving the rest empty, which looks quite odd.

**To Reproduce**
```jsx
<Carousel
        showDots={true}
        responsive={{
          desktop: {
            breakpoint: { max: 3000, min: 1024 },
            items: 3,
          },
          tablet: {
            breakpoint: { max: 1024, min: 464 },
            items: 2,
          },
          mobile: {
            breakpoint: { max: 464, min: 0 },
            items: 1,
          },
        }}
      >
      // ... several slide components/divs
</Carousel>
```
Visit on desktop or tablet sizes and click with the next button right to the end until it disappers and then click further using the remaining dots right to the active one.

**Expected behavior**
The last slide is always fixed to the right of the container.

**Screenshots**
![Bildschirmfoto 2019-06-27 um 14 28 48](https://user-images.githubusercontent.com/1384345/60266792-180fc780-98e9-11e9-8ffe-9ecaad296d36.png)

**Comment by YIZHUANG** [comment]
https://github.com/YIZHUANG/react-multi-carousel/pull/33 
Currently only supports non-infinite mode.

**Comment by tpinne** [comment]
Looks great by looking at the code changes. I would love to test and integrate this in our project. Tried to "install" the branch via yarn directly but got errors during compile. Since we are in a hot phase I didn't have time to investigate further. Is it possible to provide us with a build version of it, that we could temporarily put inside node_modules?
Or how could I test this easily using the branch?

**Comment by YIZHUANG** [comment]
Hi, 

Testing it locally.
   1. Clone the repo, and go to the branch, and run ```npm run build```.  And then run ```yarn link```
  2. In your own project, ```yarn link react-multi-carousel```

I will see if i can provide a build version.

**Comment by YIZHUANG** [comment]
@tpinne  Do ```npm install react-multi-carousel@next``` will install the test version.

**Comment by tpinne** [comment]
Works like a charm. Now this combined with `slidesToSlide` in responsive options... HEAVEN!!!

**Comment by YIZHUANG** [comment]
I planned to combined with ```slidesToSlide```after i get the infinite mode working, but currently i don't have the time to do that yet.

**Comment by tpinne** [comment]
No problem, currently we worked around that by explicitly providing the correct value to `slidesToSlide` prop on breakpoint change. So no rush for us :-)

**Comment by YIZHUANG** [comment]
The new version of dot mode is now published at version 2.0. https://github.com/YIZHUANG/react-multi-carousel/commit/5e0b3646842394e0f9c9e1367c5640954218ca67
